src/main/java/frc/robot/RobotContainer.java

```diff
@@ -29,7 +29,7 @@
   public final ShooterSubsystem shooterSubsystem = new ShooterSubsystem();
   public final IntakeSubsystem IntakeSubsystem = new IntakeSubsystem();
 
-  public final DriveCommand driveCommand = new DriveCommand(driveSubsystem);
+  public final DriveCommand driveCommand = new DriveCommand(driveSubsystem, this);
 
   public final ControllerBase mainController = new ControllerBase();
```

Comment:
So we're not using a SendableChooser to put the control map selection on the SmartDashboard?

Reply:
The switch to the intended usage of button triggers makes variable control mappings a lot harder.  It was my recommendation to go with the Button command objects at this time and we'll worry about variable mappings later.